### ISSUE TITLE
Cooling Towers are not being sequentially loaded, all towers are running

### ISSUE THREAD

**Issue by BobHenninger** [NREL/EnergyPlus#4262]
With v4.0, user test file with 10 cooling towers which were drastically oversized and programmed to load sequentially were all running and equally loaded even though 1 tower would have carried the load.  Same was happening with test file that had only 3 cooling towers.  case with only 1 cooling tower appeared to run okay.  When tested with v8.1, the 1 tower and 3 tower cases ran successfully with only 1 cooling tower running and carrying the load.  The v8.1 10 tower case CRASHED.  All cases were run for the 3-day period Aug 1-3.  v4.0 takes a long time to run (15+ minutes) but v8.1 runs much faster.
Helpdesk Ticket 1777
Test files:  
091220_U147_Asls.idf which has 10 cooling towers
091214_U147_Asls.idf which has 3 cooling towers
Hotel_Asls_091124.idf which has 1 cooling tower

**Comment by rraustad** [comment]
Looks like only the first tower operates to meet the load in V8.5.

![4262](https://cloud.githubusercontent.com/assets/5778156/15696372/c197c8a6-277d-11e6-840a-eca6f4a6b6e5.png)

**Comment by rraustad** [comment]
The 10 tower file fails on sizing in V8.5.

```
**  Fatal  ** Autosizing of cooling tower UA failed for tower COND_COOLTWR_1
```

**Comment by rraustad** [comment]
OK, all towers were grossly oversized (1E+25) leading to NAN issue and fatal on tower UA calculation.
When changing the tower capacity to be 1E+6 for each tower, only the first tower operated. When changing all towers to 1E+5, it appears the towers come on as needed to meet the load. In this figure, the first 3 towers are operating to meet the load, with other coming on line to meet additional load if necessary.

![4262-10towers2](https://cloud.githubusercontent.com/assets/5778156/15717656/f53f4ac6-27f5-11e6-84f5-36d1b8449021.png)

**Comment by Myoldmopar** [comment]
@rraustad So is there anything that needs to be done to close this issue?  Protect against insane (NaN-instigating) sized towers?

**Comment by rraustad** [comment]
I saw that the towers operated correctly when appropriately sized and moved on. I guess something is still needed here. I'll have to circle back to this when I get some time.

**Comment by Myoldmopar** [comment]
OK, if you don't anticipate getting back to this during the next few weeks I'll take it.

**Comment by rraustad** [comment]
There was a problem with sizing and is now corrected. There were two variables used to set up the RegulaFalsi routines (Par array variables) that were not set correctly and still showed -99999 or other negative number (i.e., AutoSize \* something). I ran the 10 tower defect file and see only 1 tower operating, which makes sense since all towers are grossly oversized. At least now there is no fatal or crash.

![4262towersizingandoperation](https://cloud.githubusercontent.com/assets/5778156/17896471/c6d41166-691e-11e6-9d13-b7b67daa121c.png)

**Comment by rraustad** [comment]
@EnergyArchmage while creating a unit test for single speed tower, copied the existing Merkel unit test and converted to single speed. With MyLoad = 0, the tower outlet is equal to the set point temp while the inlet is a little higher. Should all towers be off if MyLoad = 0 ... with outlet = inlet? See issue #4927 and existing unit test = CondenserLoopTowers_MerkelNoCooling.

**Comment by EnergyArchmage** [comment]
If I recall the old single speed tower controls off leaving setpoint and doesn't really respond to MyLoad, like it should.  Long standing "design" feature.

**Comment by rraustad** [comment]
See PR https://github.com/NREL/EnergyPlus/pull/5831

**Comment by mitchute** [comment]
Possibly related to #5153 ?

**Comment by rraustad** [comment]
I think #5153 might be caused by the towers not looking at MyLoad and always meeting the set point. #4927 clued me in on that subtle operation. MyLoad is not even passed into CalcSingleSpeedTower, nor to Init but a RunFlag variable is passed to Init but not used. If the tower mass flow rate is set to 0 in Init (based on RunFlag?) then the tower won't run.

**Comment by mjwitte** [comment]
Closed via #5831
